lib/lightning_web/components/new_inputs.ex

```diff
@@ -367,6 +384,82 @@ defmodule LightningWeb.Components.NewInputs do
     """
   end
 
+  def input(%{type: "tag"} = assigns) do
+    assigns =
+      assign_new(assigns, :tags, fn ->
+        case assigns[:value] do
+          tags when is_list(tags) ->
+            tags
+
+          value when is_binary(value) ->
+            value
+            |> String.split(",", trim: true)
+            |> Enum.map(&String.trim/1)
+
+          _ ->
+            []
+        end
+      end)
+
+    ~H"""
+    <div
+      id={"#{@id}-container"}
+      class="tag-input-container"
+      phx-hook="TagInput"
+      phx-feedback-for={@name}
+      data-standalone-mode={@standalone}
+    >
+      <.label :if={@label} for={@id} class="mb-2">
+        {@label}<span :if={Map.get(@rest, :required, false)} class="text-red-500"> *</span>
+        <.tooltip_for_label :if={@tooltip} id={"#{@id}-tooltip"} tooltip={@tooltip} />
+      </.label>
+
+      <small :if={@sublabel} class="mb-2 block text-xs text-gray-600">
+        {@sublabel}
+      </small>
+
+      <div class="relative">
+        <input
+          type="text"
+          id={"#{@id}_raw"}
+          placeholder={@placeholder}
+          name={"#{@name}_raw"}
+          class={[
+            "focus:outline focus:outline-2 focus:outline-offset-1 block w-full rounded-lg text-slate-900 focus:ring-0 sm:text-sm sm:leading-6",
+            "phx-no-feedback:border-slate-300 phx-no-feedback:focus:border-slate-400 disabled:cursor-not-allowed disabled:bg-gray-50 disabled:text-gray-500",
+            @errors == [] &&
+              "border-slate-300 focus:border-slate-400 focus:outline-indigo-600",
+            @errors != [] &&
+              "border-danger-400 focus:border-danger-400 focus:outline-danger-400",
+            @class
+          ]}
+          {@rest}
+        />
+        <input type="hidden" name={@name} id={@id} value={Enum.join(@tags, ",")} />
+      </div>
+      <.error :for={msg <- @errors} :if={@display_errors}>{msg}</.error>
+
+      <div class="tag-list mt-2">
+        <span
+          :for={tag <- @tags}
+          id={"tag-#{String.replace(tag, " ", "-")}"}
+          class="inline-flex items-center rounded-md bg-blue-50 p-2 text-xs font-medium text-gray-600 ring-1 ring-inset ring-gray-500/10 mr-1 my-1"
+          data-tag={tag}
+        >
+          {tag}
+          <button
+            type="button"
+            class="group relative -mr-1 h-3.5 w-3.5 rounded-sm hover:bg-gray-500/20"
+          >
+            <span class="sr-only">Remove</span>
+            <Heroicons.x_mark class="h-3 w-3 stroke-gray-600/50 group-hover:stroke-gray-600/75" />
```

Comment:
```suggestion
            <.icon name="hero-x-mark" class="h-3 w-3 stroke-gray-600/50 group-hover:stroke-gray-600/75" />
```
fyi, we moved away from rendering icons from the lib directly because it renders the whole svg

Reply:
Fixed